sdk/core/Azure.Core.TestFramework/src/Azure.Core.TestFramework.csproj

```diff
@@ -0,0 +1,18 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>$(RequiredTargetFrameworks);net461;net47</TargetFrameworks>
```

Comment:
Why are we explicitly adding net461 and net47 here?

Reply:
This was needed due to KeyVault test project - https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/keyvault/Azure.Security.KeyVault.Keys/tests/Azure.Security.KeyVault.Keys.Tests.csproj#L5

Reply:
Sigh... ok. I guess it doesn't matter much in the end for our test projects but it is a little unfortunate that we have to add this special case.